integration/client/helpers_windows.go

```diff
@@ -0,0 +1,116 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package client
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/Microsoft/hcsshim"
+	"golang.org/x/sys/windows"
+)
+
+// forceRemoveAll is the same as os.RemoveAll, but is aware of io.containerd.snapshotter.v1.windows
+// and uses hcsshim to unmount and delete container layers contained therein, in the correct order,
+// when passed a containerd root data directory (i.e. the `--root` directory for containerd).
+func forceRemoveAll(path string) error {
```

Comment:
FWIW; if we would ever need this utility outside of these integration tests, I would suggest to then move it inside a package that deals with these snapshots (perhaps to a `utils` package or `utils.go` file in https://github.com/containerd/containerd/tree/v1.6.8/snapshots/windows). That way we can import it more "focussed" where needed.

Reply:
I'm in agreeance with this; sorry I missed this change. It seems *extremely* snapshotter focused and not very general purpose so +1 from me

Reply:
Thanks for looking! Yes, so currently these utilities are only used in the integration tests, so I didn't want to give them a larger "audience" than possible by moving them into the `snapshots/windows` package. My comment was mostly for _IF_ there would be a demand for that.

And countering myself; we should be careful with creating a `utils` package (or at least consider if we want to); naming a package `utils` is commonly discouraged, as it's too easy to become a kitchen-sink of "I don't know where to put it, just dump it in "utils". And before we know if, we created a new "sys" package with too many dependencies that's imported in too many places 😂

Reply:
Oh double comment after the fact on an already merged PR.. I was actually referring to agreeing with the entire change not the utils comment specifically LOL I realized I'm commenting on a specific comment here